CODEOWNERS

```diff
@@ -5,3 +5,6 @@
 /webapp/package-lock.json @mattermost/web-platform
 /webapp/platform/*/package.json @mattermost/web-platform
 /webapp/scripts @mattermost/web-platform
+/server/channels/db/migrations @mattermost/server-platform
```

Comment:
This error shows up because this team may not have "write" permission. 
check https://community.mattermost.com/private-core/pl/ae8b9ato13g95f1otxkq594iow

Reply:
Thanks, I am aware :) We will get it fixed.